pkg/skaffold/schema/latest/config.go

```diff
@@ -191,6 +191,10 @@ type LocalBuild struct {
 	// connects to a remote cluster.
 	Push *bool `yaml:"push,omitempty"`
 
+	// TryImportMissing should attempt to import this artifact from
+	// Docker (either locally or a remote registry) if not in cache.
+	TryImportMissing bool `yaml:"tryImportMissing,omitempty"`
```

Comment:
Can you please make this a pointer like other?
And then you would need to set to it false in `defaultToLocalBuild` in `pkg/skaffold/schemadefaults/`